src/field_theory/subfield.lean

```diff
@@ -98,27 +104,43 @@ lemma mem_carrier {s : subfield K} {x : K} : x ∈ s.carrier ↔ x ∈ s := iff.
 equalities. -/
 protected def copy (S : subfield K) (s : set K) (hs : s = ↑S) : subfield K :=
 { carrier := s,
-  inv_mem' := hs.symm ▸ S.inv_mem',
+  inv_mem'  := hs.symm ▸ S.inv_mem',
+  mul_comm' := hs.symm ▸ S.mul_comm',
   ..S.to_subring.copy s hs }
 
 @[simp] lemma coe_to_subring (s : subfield K) : (s.to_subring : set K) = s :=
 rfl
 
-@[simp] lemma mem_mk (s : set K) (ho hm hz ha hn hi) (x : K) :
-  x ∈ subfield.mk s ho hm hz ha hn hi ↔ x ∈ s := iff.rfl
+@[simp] lemma mem_mk (s : set K) (ho hm hz ha hn hi hc) (x : K) :
+  x ∈ subfield.mk s ho hm hz ha hn hi hc ↔ x ∈ s := iff.rfl
 
 @[simp] lemma mem_to_subring (s : subfield K) (x : K) :
   x ∈ s.to_subring ↔ x ∈ s := iff.rfl
 
+/-- A constructor for subfields of fields:
+  the commutativity axiom is derived from the ambient field. -/
+def mk' {K : Type*} [field K] (S : subring K) (hi : ∀ x ∈ S, x⁻¹ ∈ S) :
```

Comment:
What's the point of this new definition? Isn't it identical to the old `subring.to_subfield`?

Reply:
Hah, I guess you are right. I'll drop it (and the other one that @Vierkantor pointed out below).